corehq/apps/hqwebapp/static/cloudcare/less/formplayer-webapp/form.less

```diff
@@ -80,6 +80,14 @@
     }
   }
 
+  .group-border {
```

Comment:
From a style point we probably want this to looks the same as the border around grouped tile fields. Therefore I would suggest to use a common style between the two of them. At least for the `border` and `border-radius` attributes. This is especially relevant in light of the interest to style commcare by the app builders.

I am not sure if a separate class would be better suited or variables like the colors we define in `variables.less`. @biyeun Not sure if you have any thoughts on that?

Reply:
in bootstrap 5 stylesheet, I would use cards to solve this rather than creating a new component which basically does what cards does. You could use a well for the time being, or this class and then make a note about updating it in the migration.

see cards docs here:
https://getbootstrap.com/docs/5.3/components/card/

Reply:
I think a bootstrap 3 panel would be closer to what we want than a well. But for grouping of fields on case tiles at least there was a desire to configure a border and/or background shading. Not sure if it is to late to argue against that flexibility.